libgraph/include/katana/PropertyGraph.h

```diff
@@ -98,7 +98,9 @@ class KATANA_EXPORT PropertyGraph {
   using EntityTypeIDArray = katana::NUMAArray<EntityTypeID>;
 
   PropertyGraph(PropertyGraph&& other) = default;
+  PropertyGraph& operator=(PropertyGraph&& other) = default;
 
+public:
```

Comment:
nit: looks like an extra `public` declaration snuck in during the rebase

Reply:
I will fix this next time I touch this code. I have a hunch I'll need to touch it a little in the transformed view wrapping coming soon.